beacon-chain/blockchain/service.go

```diff
@@ -62,6 +62,7 @@ type Service struct {
 	clockWaiter          startup.ClockWaiter
 	syncComplete         chan struct{}
 	blobNotifiers        *blobNotifierMap
+	blockBeingSynced     *currentlySyncingBlock
```

Comment:
It is a little confusing that there's a bool-returning function called the same, when here we have a list of blocks.